src/instructlab/model/backends/backends.py

```diff
@@ -167,20 +167,24 @@ def is_model_gguf(model_path: pathlib.Path) -> bool:
     # Third Party
     from gguf.constants import GGUF_MAGIC
 
-    with open(model_path, "rb") as f:
-        # Memory-map the file on the first 4 bytes (this is where the magic number is)
-        mmapped_file = mmap.mmap(f.fileno(), length=4, access=mmap.ACCESS_READ)
+    try:
+        with open(model_path, "rb") as f:
+            # Memory-map the file on the first 4 bytes (this is where the magic number is)
+            mmapped_file = mmap.mmap(f.fileno(), length=4, access=mmap.ACCESS_READ)
 
-        # Read the first 4 bytes
-        first_four_bytes = mmapped_file.read(4)
+            # Read the first 4 bytes
```

Comment:
Can you make sure this is not a space/tab issue and stick with spaces? Thanks

Reply:
when I edit the file, changing to spaces results in no changes to the file to commit, same goes for tabs, unsure how to fix how this is rendering